.github/workflows/client-code-checks.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+    branches:
+      - main
+      - develop
+      # This pattern matches branches for release candidates (e.g., v2.6.0-RC)
+      - v*-RC
```

Comment:
Supports cases for RC branches. This workflow will run against any PRs that are open and targeting these branches.

As of the time this was written, the GitHub Action is failing because v2.6.0-RC has a console log.

Reply:
Why not on every PR ?

Reply:
Thats a good question actually, it should just be applied to every PR. Will change soon